tests/cases/fourslash/quickInfoTypedefTag.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path='fourslash.ts' />
+// @allowJs: true
+// @Filename: a.js
+//// /**
+////  * The typedef tag should not appear in the quickinfo.
+////  * @typedef {{ foo: 'foo' }} Foo
+////  */
+//// function f() { }
+//// f/*1*/()
+//// /**
+////  * A removed comment
+////  * @tag Usage shows that non-param tags in comments explain the typedef instead of using it
+////  * @typedef {{ nope: any }} Nope not here
+////  * @tag comment 2
+////  */
+//// function g() { }
+//// g/*2*/()
+//// /**
+////  * The whole thing is kept
```

Comment:
There are no longer cases where we drop part of a comment, correct?

Reply:
That's right.